packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -238,63 +242,73 @@ export let PageHeader = React.forwardRef(
         }
       }
 
-      setComponentCssCustomProps((prevCSSProps) => {
-        return {
-          ...prevCSSProps,
-          [`--${blockClass}--max-action-bar-width-px`]: newActionBarWidth,
-          [`--${blockClass}--button-set-in-breadcrumb-width-px`]: `${newPageActionInBreadcrumbWidth}`,
-        };
+      utilSetCustomCSSProps(headerRef, {
+        [`--${blockClass}--max-action-bar-width-px`]: newActionBarWidth,
+        [`--${blockClass}--button-set-in-breadcrumb-width-px`]: `${newPageActionInBreadcrumbWidth}`,
       });
     }, [
       actionBarColumnWidth,
       actionBarMaxWidth,
       actionBarMinWidth,
       pageActionInBreadcrumbMaxWidth,
       pageActionInBreadcrumbMinWidth,
+      headerRef,
     ]);
 
+    const doUpdateBreadcrumbPosition = (scrollY) => {
```

Comment:
```suggestion
    // Update the breadcrumb position in response to vertical scroll.
    // This can include flipping between sticky and relative position.
    const doUpdateBreadcrumbPosition = (scrollY) => {
```